docs/netobserv_cli.adoc

```diff
@@ -0,0 +1,69 @@
+// Automatically generated by './scripts/generate-doc.sh'. Do not edit, or make the NETOBSERV team aware of the editions.
+:_mod-docs-content-type: REFERENCE
+[id="network-observability-cli-usage_{context}"]
+= Network Observability CLI usage
+
+Users of Netobserv CLI can pass command line arguments to enable features or pass configuration options to the eBPF agent and flowlogs-pipeline.
+This document lists all the supported command line options and their possible values.
+
+== Flows usage
+[cols="1,1,1",options="header"]
+|===
+| Option | Description | Default
+|--enable_pktdrop|  enable packet drop                  | false
+|--enable_dns|      enable DNS tracking                 | false
+|--enable_rtt|      enable RTT tracking                 | false
```

Comment:
Currently, we have these called out as enrichment options along with `interfaces` and `help`. Maybe that distinction isn't necessary, but I wanted to point it out and discuss. https://docs.openshift.com/container-platform/4.12/observability/network_observability/netobserv_cli/netobserv-cli-reference.html#network-observability-cli-enrichment_netobserv-cli-reference

Reply:
I'm not sure it's mandatory to make the distinction between filters and features (pktdrop / dns / rtt) but I can improve the script in that way if you feel it was better.

Interfaces is a filter only available for flows so it has been moved to the proper place.
`--help` option has been replaced by `help` sub command as `oc netobserv flows help`

Reply:
Okay, if it isn't mandatory then we don't have to make that distinction.